### PR TITLE
IDCs: Adding functionality to not group pads near the sector edges

### PR DESCRIPTION
**Pull request by matthias-kleiner**
other changes
- [drawing grouping information on debug plots](https://github.com/AliceO2Group/AliceO2/files/7641329/grouping_rows_2_2_2_3_3_3_2_3_2_2_pads_5_6_7_9_4_5_6_7_11_16_rowThr_3_3_3_3_2_2_2_2_2_2_padThr_1_1_1_1_1_1_1_1_1_1_ovRows-0_ovPads-0.pdf)
- Moving helper function for setting grouping parameters from string
- Renaming members in struct (coding conventions)

### CODE REVIEW

File: Detectors/TPC/calibration/src/IDCGroupingParameter.cxx [hunk 1]
```diff
@@ -10,7 +10,65 @@
...
+  std::vector<unsigned char> vgroupLastPadsThreshold;
+  vgroupPads.reserve(Mapper::NREGIONS);
+  vgroupRows.reserve(Mapper::NREGIONS);
+  vgroupLastRowsThreshold.reserve(Mapper::NREGIONS);
+  vgroupLastPadsThreshold.reserve(Mapper::NREGIONS);
+
+  std::transform(tgroupPads.begin(), tgroupPads.end(), std::back_inserter(vgroupPads), &boost::lexical_cast<int, std::string>);
+  std::transform(tgroupRows.begin(), tgroupRows.end(), std::back_inserter(vgroupRows), &boost::lexical_cast<int, std::string>);
+  std::transform(tgroupLastRowsThreshold.begin(), tgroupLastRowsThreshold.end(), std::back_inserter(vgroupLastRowsThreshold), &boost::lexical_cast<int, std::string>);
+  std::transform(tgroupLastPadsThreshold.begin(), tgroupLastPadsThreshold.end(), std::back_inserter(vgroupLastPadsThreshold), &boost::lexical_cast<int, std::string>);
```
Comment (wiechula) [inline Detectors/TPC/calibration/src/IDCGroupingParameter.cxx]: I think you can use the `RangeTokenizer` (https://github.com/AliceO2Group/AliceO2/blob/dev/Algorithm/include/Algorithm/RangeTokenizer.h):
```c++
#include "Algorithm/RangeTokenizer.h"
const auto arr = o2::RangeTokenizer::tokenize<unsigned char>("1,2,3,4,5");
arr
(std::vector<unsigned char, std::allocator<unsigned char> > &) { '1', '2', '3', '4', '5' }
```

**Review by wiechula — COMMENTED** [review]
Very nice, just one comment you might want to take into account.

File: Detectors/TPC/calibration/src/IDCGroupingParameter.cxx (hunk 1 above)
Comment (matthias-kleiner) [reply Detectors/TPC/calibration/src/IDCGroupingParameter.cxx]: Thanks for the comment. I switched to the RangeTokenizer as suggested

**Review by wiechula — APPROVED** [review]

**Comment by wiechula** [comment]
@davidrohr , can this be merged?

**Pull request merged**